dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
It's like a linter to anyone that want to update the `Dockerfile` but they could still do it manually.